travis-ci/vmtest/helpers.sh

```diff
@@ -2,11 +2,15 @@
 # $2 - fold identifier, no spaces
 # $3 - fold section description
 travis_fold() {
-	local YELLOW='\033[1;33m'
-	local NOCOLOR='\033[0m'
-	echo travis_fold:$1:$2
```

Comment:
removing travis_fold echo will break travis logs, no?

Reply:
argh, you right